pimd/pim6_cmd.c

```diff
@@ -1919,10 +1926,167 @@ DEFPY (show_ipv6_mroute_summary_vrf_all,
 	return CMD_SUCCESS;
 }
 
+DEFPY (debug_pimv6,
+       debug_pimv6_cmd,
+       "[no] debug pimv6",
```

Comment:
Hm. Every other place we have `ipv6 pim …` — should we maybe do `debug ipv6 pim …` for all these commands?

(Just asking, not particularly set on it…)

Reply:
Yeah I too thought of this but FRR v4 has debug pim and not debug ip pim, so kept it this way to keep it consistent with pimv4.